tests/unit/Codeception/Util/AnnotationTest.php

```diff
@@ -92,4 +92,63 @@ public function testSingleLineAnnotation()
                 ->method('testSingleLineAnnotation')
                 ->fetch('value'));
     }
+
+    public function testFetchAllExamples()
+    {
+        $class = new class {
+            /**
+             * @example ["example 1/2"]
+             * @example ["example 2/2"]
+             */
+            public function multipleAnnotations() {}
+
+            /**
+             * @example ["example 1/1"]
+             */
+            public function singleAnnotation() {}
+
+            #[\Codeception\Attribute\Examples('example 1/2')]
+            #[\Codeception\Attribute\Examples('example 2/2')]
+            public function multipleAttributes() {}
+
+            #[\Codeception\Attribute\Examples('example 1/1')]
+            public function singleAttribute() {}
+        };
+
+        $this->assertSame(
+            ['["example 1/2"]', '["example 2/2"]'],
+            Annotation::forMethod($class, 'multipleAnnotations')->fetchAll('example'));
+
+        $this->assertSame(
+            ['["example 1/1"]'],
+            Annotation::forMethod($class, 'singleAnnotation')->fetchAll('example'));
+
+        $this->assertSame(
+            [['example 1/2'], ['example 2/2']],
+            Annotation::forMethod($class, 'multipleAttributes')->fetchAll('example'));
+
+        $this->assertSame(
+            [['example 1/1']],
```

Comment:
`phpcs` doesn't like style of this file

Reply:
Sorry, my bad. Missed PHPCS checks. Fixed.